retrofit-adapters/rxjava2/src/test/java/retrofit2/adapter/rxjava2/RxJava2CallAdapterFactoryTest.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
```

Comment:
Wasn't sure whether it should be 2015 or 2016. Sticked to 2015 since that's also the date from `RxJavaCallAdapterFactoryTest`